source/base/multithread_info.cc

```diff
@@ -134,6 +145,19 @@ MultithreadInfo::initialize_multithreading()
   done = true;
 }
 
+#ifdef DEAL_II_WITH_CPP_TASKFLOW
+tf::Executor &
+MultithreadInfo::get_taskflow_executor()
+{
+  Assert(
+    executor.get() != nullptr,
+    ExcMessage(
+      "Please initialize multithreading using MultithreadInfo::set_thread_limit() first."));
```

Comment:
Do we need to put that into every program now to make this work?

Reply:
@bangerth No, it will be automatically called by our static `MultithreadInfo::initialize_multithreading()` initializer (end of `multithread_info.cc`).

Reply:
answered in a comment right above, now.